projects/react/README.md

```diff
@@ -0,0 +1,22 @@
+# @maskito/react
+
+[![npm version](https://img.shields.io/npm/v/@maskito/react.svg)](https://npmjs.com/package/@maskito/react)
+[![npm bundle size](https://img.shields.io/bundlephobia/minzip/@maskito/react)](https://bundlephobia.com/result?p=@maskito/react)
+
+<p align="center">
+    <img src="https://raw.githubusercontent.com/Tinkoff/maskito/main/projects/demo/src/assets/icons/maskito.svg" alt="Maskito logo" height="120px">
+</p>
+
+<p align="center">
+    <a href="https://tinkoff.github.io/maskito/angular">Documentation</a> •
```

Comment:
Let's temporarily modify in this way.
In the future (when documentation for react is ready), we will change it again.
```suggestion
    <a href="https://tinkoff.github.io/maskito">Documentation</a> •
```

Reply:
done